backend/src/controllers/new-cars/new-cars.ts

```diff
@@ -0,0 +1,20 @@
+import { newerCars } from '@helpers/new-cars/new-cars-helper.ts';
+import httpStatus from 'http-status-codes';
+
+import type { Response, NextFunction, Request } from 'express';
+
+const newCars = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): Promise<void> => {
+  try {
+    const newCars = await newerCars(4);
+    res.status(httpStatus.OK).json(newCars);
```

Comment:
1. Why did you import the `service` method from the `helpers` directory? 
2. Please look at how the services are used in other controllers. They are imported as `import * as someService from '@services/blablabla`.
3. Name of the functions is misleading. Functions **must be based on the _verb_** (get, add, fetch, replace, ...). `newer` is the _adverb_ that doesn't convey function desires. So it would be much better to be named `getLastCreatedCars(amount: number)`